argo/src/Argo/DefaultMain.hs

```diff
@@ -284,14 +287,20 @@ maxOccupancyOpt =
    (Opt.long "max-occupancy" <>
     Opt.help "Maximum number of active sessions allowed at once." <>
     Opt.showDefault <>
-    Opt.value 10))
+    Opt.value 1))
```

Comment:
Does this mean that a server only saves one previous state now? Or if we go by the description immediately above, does this mean that a server can only connect to a single client at a time now? Neither of these seem to me like something we want.

Reply:
Yes, it essentially means by _default_ the server will connect to at most one client simultaneously (a new client connection which initiates some computation would cause the previous client's state on the server to be evicted). It can be customized at the command line (there's some tests that do that, e.g., upping it to `2` and then having their connections interact via interrupts etc).

I chose `1` as I believe this is the most common (only?) use case in practice from what I've observed thus far, and it will keep the server's memory footprint small for that use case. Happy to change it if we think it's a problem.

It might also be nice at some point to make this a dynamic server option as well... but again, it's hard to know based on our limited observed use cases. The impression I get is the server is being used exclusively for single connections performing linear series of computations, so I defaulted it to be optimized for that with knobs for other desires.

Reply:
Yeah, I think it's reasonable to have the default be `1`.